pack_and_terraform.bash

```diff
@@ -125,6 +137,10 @@ if echo "$RUNTF" | grep -iq "^y"; then
     echo "You need to create a certificate in AWS Certificate Manager with imported certs or the admin for the site need to approve aws create it."
     read -p "This needs to be done to make following process working. Done? [y/n] " CONFIGURED_CERT
 
+    if [ -z "$AWS_CERT_NAME" ]; then
+        read -p "Enter the certificate name: " AWS_CERT_NAME
```

Comment:
can you change the description to be more clear like `Enter the SSL/TLS certificate's domain name` and also update the readme [here](https://github.com/uc-cdis/cloud-automation#configure-certificate-for-your-domain) to mention it can be either exact match or the parent domain name?